### PR TITLE
Creating a contribution control checklist

### PR DESCRIPTION
**Pull request by Sirlupinwatson1**
What you think about a contribution checklist. It Could be benefic for both the actors and the readers

### CODE REVIEW

**Comment by Sirlupinwatson1** [comment]
This Checklist would need to be reviewed of course before setting it up online. 

It would need to have dinstincts steps to show how to do it. Like the usual doc

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+
+
+**Sections**
+
+1- Starting Point
+
+2- Integration
+
+3- Levelling up
```
Comment (MMajed001) [inline CONTRIBUTING.md]:
```suggestion
3- Levelling up
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+
+
+
+
+**Sections**
+
+1- Starting Point
+
+2- Integration
```
Comment (MMajed001) [inline CONTRIBUTING.md]:
```suggestion
2- Integration
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (Sirlupinwatson1) [reply CONTRIBUTING.md]: So here is my first real test huh, Let's do solve it

Comment (Sirlupinwatson1) [reply CONTRIBUTING.md]: I try to achieve something, but not going the way i want it to go haha. But it will eventually.

Comment (Sirlupinwatson1) [reply CONTRIBUTING.md]: Ok so i did manage to work a little, but i had hard time to find myself . I should be good to go for further work now. I need to go right now but i be back working later  tonight.... 

I wanted to do some fantasy and impress you, but well... I will do some later you can be 100 % sure about it

**Comment by MMajed001** [comment]
Sirlupinwatson1 yesterday Author
Ok so i did manage to work a little, but i had hard time to find myself . I should be good to go for further work now. I need to go right now but i be back working later tonight....

I wanted to do some fantasy and impress you, but well... I will do some later you can be 100 % sure about it

**Comment by MMajed001** [comment]
Sirlupinwatson1 yesterday Author
Ok so i did manage to work a little, but i had hard time to find myself . I should be good to go for further work now. I need to go right now but i be back working later tonight....

I wanted to do some fantasy and impress you, but well... I will do some later you can be 100 % sure about it

**Pull request closed without merging**